### ISSUE TITLE
Compiling Error on cargo build nova-snark 0.37.0

### ISSUE THREAD

**Issue by wanglu4042** [microsoft/Nova#335]
Hi， when I try to build nova-snark 0.37.0, I met this error.

```
The following warnings were emitted during compilation:

warning: pasta-msm@0.1.4: Compiler family detection failed since it does not define `__clang__`, `__GNUC__`, `__EMSCRIPTEN__` or `__VXWORKS__`, also does not accept cl style flag `-?`, fallback to treating it as GNU
warning: pasta-msm@0.1.4: Compiler family detection failed due to error: ToolFamilyMacroNotFound: Expects macro `__clang__`, `__GNUC__` or `__EMSCRIPTEN__`, `__VXWORKS__` or accepts cl style flag `-?`, but found none
warning: pasta-msm@0.1.4: cl: ������ error D8021 :��Ч����ֵ������/Wno-unused-function��

error: failed to run custom build command for `pasta-msm v0.1.4`

Caused by:
  process didn't exit successfully: `D:\ZKP_file\Work\test\Nova-main\target\debug\build\pasta-msm-84791e421c89768d\build-script-build` (exit code: 1)
  --- stderr


  error occurred: Command "nvcc" "-ccbin=C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.29.30133\\bin\\HostX64\\x64\\cl.exe" "-Xcompiler" "-nologo" "-Xcompiler" "-MD" "-G" "-Xcompiler" "-Z7" "-Xcompiler" "-Brepro" "-I" "C:\\Users\\WANGLU\\.cargo\\registry\\src\\index.crates.io-6f17d22bba15001f\\semolina-0.1.4\\src" "-I" "C:\\Users\\WANGLU\\.cargo\\registry\\src\\index.crates.io-6f17d22bba15001f\\sppark-0.1.10\\sppark" "-Xcompiler" "-W4" "-Xcompiler" "-Wno-unused-function" "-arch=sm_70" "-DTAKE_RESPONSIBILITY_FOR_ERROR_MESSAGE" "-D__ADX__" "-o" "D:\\ZKP_file\\Work\\test\\Nova-main\\target\\debug\\build\\pasta-msm-42d2b2a4dd80c16c\\out\\0c0a7217c765217b-pallas.o" "-c" "cuda/pallas.cu" with args nvcc did not execute successfully (status code exit code: 2).
```
My cude version is 11.5
```
(base) PS D:\ZKP_file\Work\test\Nova-main> nvcc -V 
nvcc: NVIDIA (R) Cuda compiler driver Copyright (c) 2005-2021 NVIDIA Corporation Built on Mon_Sep_13_20:11:50_Pacific_Daylight_Time_2021 Cuda compilation tools, release 11.5, V11.5.50 Build cuda_11.5.r11.5/compiler.30411180_0
And VS2019 is also installed.
```
Could you help me with this error?

**Comment by srinathsetty** [comment]
This appears to be an issue with: https://github.com/supranational/pasta-msm/. Have you considered filing an issue there?

**Comment by srinathsetty** [comment]
Closing it for now since I see a related issued opened with pasta-msm. Please open a new issue if this is still an issue on the Nova repository side.

**Comment by srinathsetty** [comment]
This is now fixed with an update #342. Once merged, we will push a new version.
